Fakebook.Profile/Fakebook.Profile.UnitTests/DomainTests/ProfileModelTests.cs

```diff
@@ -283,5 +285,41 @@ public void EarlierTodayIsValid()
             Assert.Equal(earlierToday, profile.BirthDate);
         }
         #endregion
+
+        /// <summary>
+        /// Check that follower emails have the correct format.
+        /// </summary>
+        [Fact]
+        public void TestAddFollowerEmail() 
+        {
+            // arrange
+            DomainProfile profile = new("emailtest@gmail.com", "Bob", "Fields");
+            string followerEmail = "testermail@gmail.com";
+
+            // act
+            profile.AddFollower(followerEmail);
+
+            // assert
+            Assert.Contains(followerEmail, profile.FollowerEmails);
+
+        }
+
+        /// <summary>
+        /// Check that following emails have the correct format.
+        /// </summary>
+        [Fact]
+        public void TestAddFollowingEmail()
+        {
+            // arrange
+            DomainProfile profile = new("emailtest@gmail.com", "Bob", "Fields");
+            string followEmail = "testermail@gmail.com";
+
+            // act
+            profile.AddFollow(followerEmail);
```

Comment:
Make sure this argument name is matching the variable name you are creating above!